com.vscodium.codium-insiders.yaml

```diff
@@ -0,0 +1,145 @@
+      - rm -r codium.deb control.tar.* data.tar.xz debian-binary usr
+      - mkdir /app/tools
+    sources:
+      - type: file
+        url: https://github.com/VSCodium/vscodium-insiders/releases/download/1.85.0.23310-insider/codium-insiders_1.85.0.23310_amd64.deb
+        sha256: 7b4b90bdb74d7761c4c7ad4f66b189f221d8fd8bff4b1df15cf2b2fe9182aa60
+        dest-filename: codium.deb
+        only-arches:
+          - x86_64
+        x-checker-data:
+          type: json
+          url: https://api.github.com/repos/VSCodium/vscodium-insiders/releases/latest
+          tag-query: .tag_name
+          version-query: .tag_name
+          timestamp-query: .published_at
+          url-query: '[.assets[] | select(.name|test("codium-insiders_.+_amd64.deb$"))][0]
+            | .browser_download_url'
+      - type: file
+        url: https://github.com/VSCodium/vscodium-insiders/releases/download/1.85.0.23310-insider/codium-insiders_1.85.0.23310_arm64.deb
+        sha256: 44b9331333c94b984ac510d2b7436168c91e4899d226f0edd5d082b3d1da4747
+        dest-filename: codium.deb
+        only-arches:
+          - aarch64
+        x-checker-data:
+          type: json
+          url: https://api.github.com/repos/VSCodium/vscodium-insiders/releases/latest
+          tag-query: .tag_name
+          version-query: .tag_name
+          timestamp-query: .published_at
+          url-query: '[.assets[] | select(.name|test("codium-insiders_.+_arm64.deb$"))][0]
+            | .browser_download_url'
+      - type: file
+        url: https://raw.githubusercontent.com/VSCodium/vscodium/f1fa60db6d5332bef6cb2fad55b809a48b76179c/icons/insider/codium_cnl.svg
+        sha256: 0cc2ded6ad8109c59bdfad9ef71c0afda0e6d5d6a26aaf03eac5d4a8a92fa4e5
+        dest-filename: codium-insiders.svg
+      - type: file
+        path: com.vscodium.codium-insiders.metainfo.xml
+  - name: zypak
+    sources:
+      - type: git
+        url: https://github.com/refi64/zypak.git
+        branch: main
+        commit: ded79a2f8a509adc21834b95a9892073d4a91fdc
+  - name: ide-flatpak-wrapper
+    buildsystem: meson
+    config-opts:
+      - -Deditor_binary=/app/share/codium-insiders/codium-insiders
+      - >-
+        -Deditor_args=[
+        '/app/share/codium-insiders/resources/app/out/cli.js',
+        '--ms-enable-electron-run-as-node',
+        '--extensions-dir', '$XDG_DATA_HOME/codium-insiders/extensions'
+        ]
+      - -Dprogram_name=codium-insiders
+      - -Deditor_title=VSCodium Insiders
+      - -Dzypak=enabled
+      - -Dzypak_args=[ 'host', '-' ]
+      - >-
+        -Denvs=[
+        "ELECTRON_RUN_AS_NODE=1"
+        ]
+      - >-
+        -Denvs_inner=[
+        "ZYPAK_SPAWN_LATEST_ON_REEXEC=0",
+        ]
+      - -Dfirst_run_template=README.md
+      - -Dsdk_update_template=sdk_update.md
+      - -Dflagfile_prefix=flatpak-vscodium-insiders
+    sources:
+      - type: git
+        url: https://github.com/flathub/ide-flatpak-wrapper.git
+        branch: zypak
```

Comment:
```suggestion
```
Same as above

Reply:
In this the best is to attach to branch instead, then I removed the commit line

Reply:
no. using branch is NOT allowed. It prevent from producing build, or even keeping them working.

Reply:
this should be a commit or a tag